source/val/validate_atomics.cpp

```diff
@@ -28,6 +28,24 @@
 namespace spvtools {
 namespace val {
 
+static bool IsStorageClassAllowedByUniversalRules(uint32_t storage_class) {
```

Comment:
Please drop the static and put this function in an anonymous namespace to match the style of the rest of the validator.

Reply:
Donw